messagingpush/src/main/java/io/customer/messagingpush/CustomerIOPushNotificationHandler.kt

```diff
@@ -240,11 +240,14 @@ internal class CustomerIOPushNotificationHandler(
         )
     }
 
-    private fun addImage(
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    internal fun addImage(
         imageUrl: String,
         builder: NotificationCompat.Builder,
         body: String
     ) = runBlocking {
+        if (imageUrl.isEmpty()) return@runBlocking
```

Comment:
I think we should avoid running into `runBlocking` when the URL is null or empty. We can exit early without needing to involve the code in `runBlocking`.

Optional suggestion: I would prefer if we had a single `isNullOrBlank` check for `imageUrl` instead of checking for `null` before calling this method and then for `empty` inside the method. I'm okay with moving the check either before calling this or inside the method.